src/main/resources/edu/oregonstate/mist/api/resource.properties

```diff
@@ -1,23 +1,21 @@
 # Response Message Properties
 
 ## Bad Request
-badRequest.userMessage = Bad Request - the application submitted invalid data. Please try again.
+badRequest.title = Bad Request - the application submitted invalid data. Please try again.
```

Comment:
Like we discussed in person, could we make the `title` to `Bad Request` and have the default `detail` field be the `the application submitted invalid data. Please try again.`

> * title: a short, human-readable summary of the problem that SHOULD NOT change from occurrence to occurrence of the problem, except for purposes of localization.
> * detail: a human-readable explanation specific to this occurrence of the problem. Like title, this field’s value can be localized.

http://jsonapi.org/format/#errors